starter-code/src/functions-and-arrays.js

```diff
@@ -1,15 +1,49 @@
 // Iteration #1: Find the maximum
+console.log(Math.max(3, 2));
+//or
+function maxTowNumbers(n1, n2){
```

Comment:
Here the name is incorrect. I see that jasmine failed a lot of your functions because of naming issues it seems.
Did you check Jasmine if you were psasing the exercises?